openpype/pipeline/frames.py

```diff
@@ -0,0 +1,62 @@
+"""Utility functions.
+
+This module contains utility functions dedicated to operations
+"""
+
+
+from openpype.lib import Logger
+
+log = Logger.get_logger(__name__)
+
+
+def get_frame_start_str(frame_start, frame_end):
+    """Get frame start string.
+
+    Args:
```

Comment:
I feel like it's quite unclear what this function does reading the docstring.

Basically what it does is return the start frame with padding so that it matches the length of the end frame.
Which in essence would be as simple as:
```python
def get_frame_start_str(frame_start, frame_end):
    padding = len(str(frame_end))
    return str(frame_start).zfill(padding)
```

Example:
```python
print(get_frame_start_str(10, 100))
# 010
```

Reply:
thanks here it is added as you suggested 859c9ac139d7393757852481ac1be7c9610b7c33